style.css

```diff
@@ -0,0 +1,166 @@
+body {
+  background-color: black;
+  display: flex;
+  max-width: 1024px;
+  margin: 0 auto;
+  flex-direction: column;
+}
+img {
+  width: 35%;
+  height: 80%;
+  order: +2;
+}
+.flexHeader {
+  display: flex;
+  justify-content: space-between;
+}   
+.info {
+  display: flex;
+  flex-direction: column;   
+}
+.object {
+  flex-basis: 100%; width: max-content;
+  background: rgba(105,105,105,0.3);
+  padding: 10px;
+  font-size: 30px;
+  margin-top: 0;
+}
+main {
+  background-color: white;
+  padding: 5% 5%;
+}
+.sel {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 15px;
+}
+.name {
+  font-size: 70px;
+  margin-top: 0;
+}
+.contact {
+  font-size: 40px;
+  margin: 0;
+}
+.connection {
+  font-size: 20px;
+  text-decoration: none;
+}
+.connection p a {
+  text-decoration: none;
+  color: black;
+}
+.connection p a:hover {
+  color: magenta;     
+}
+.connection p:hover {
+  transform: translateX(5px); 
+}
+.word {
+  font-weight: bold;
+}
+.about-me {
+  font-size: 20px;
+}
+.info-about-me {
+  margin: 0px;
+  position: relative;
+}
+.lang p , .skills p , .education p {
+  font-size: 20px;
+  margin: 0px;
+}
+.collor {
+  background: green;
+  display: inline;
+  color: honeydew;
+}
+.info-about-me::before {
+  content: '';
+  height: 100%;
+  width: 10px;
+  background: seagreen;
+  position: absolute;
+  left: -5.55%;
+}
+@media(max-width: 1024px) { 
+  .sel {
+    display: flex;
+    flex-wrap: wrap;
+  }
+}
+@media (max-width: 972px) {
+  img {
+    width: 30%;
+    height: 75%;
+  }
+}
+@media (max-width: 929px) {
+  img {
+    width: 27%;
+    height: 72%;
+  }
+}
+@media (max-width: 867px) {
+  img {
+    width: 24%;
+    height: 71%;
+  }
+}
+@media (max-width: 833px) {
+  .name {
+    font-size: 60px;
+  }
+}
+@media (max-width: 754px) {
+  .name {
+    font-size: 50px;
+  }
+  .object {
+    font-size: 24px;
+  }
+}
+@media (max-width: 615px) {
+  .name {
+    font-size: 40px;
+  }
+  .object {
+    font-size: 20px;
+  }
+  .contact {
+    font-size: 30px;
+  }
+}
+@media (max-width: 524px) {
+  .flexHeader {
+    display: flex;
```

Comment:
Display is already flex (https://github.com/Mikita-Davidovich/rsschool-cv/pull/2/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR14). You don't need to repeat it